packages/ts/react-crud/src/autogrid-columns.tsx

```diff
@@ -16,10 +16,11 @@ import {
   NumberHeaderFilter,
   StringHeaderFilter,
   TimeHeaderFilter,
+  type HeaderFilterProps,
 } from './header-filter';
 import type { PropertyInfo } from './property-info';
 
-export type ColumnOptions = Omit<GridColumnProps<any>, 'dangerouslySetInnerHTML'>;
+export type ColumnOptions = HeaderFilterProps & Omit<GridColumnProps<any>, 'dangerouslySetInnerHTML'>;
```

Comment:
It's inconvenient that you need to configure a debounce individually for every single column. It seems less likely that you want a debounce for individual columns instead of for all of them. Same for the minimum length maybe.

So maybe it would make more sense to start with a version that allows to configure this for the whole grid?

Reply:
I see three levels we could allow users to configure: 
1. whole grid -  one property would apply to all of the filters
2. type specific - apply debounce only for string filters or change placeholder only for number type filters
3. column specific - current solution, configure each column separately. 

For now, I implemented 3rd option as it was possible without any significant changes.
Grid level is also possible without any significant changes. 
The type level could require some bigger changes.

I don't see a debounce option for types other than number and string; maybe I am wrong.
This also could be a good start, or I can add props to both grid and column levels. 

I was also deciding if we want to have to debounce by default but I decided against it as it makes default filtering feel a bit more sluggish. 

As for minimum length - this is a quite specific requirement - usually for performance reasons. 
I would leave it as column-specific.

Reply:
After internal talk, I will leave column-specific filter properties. We can return to this in the future.

Also will implement the default value for debounce (200ms) as in webcomponents: 
https://github.com/vaadin/web-components/blob/6f4bc1e642db3cc2781146af3a7e131c682e8c0b/packages/grid/src/vaadin-grid-filter-element-mixin.js#L98C101-L98C101